NTO/OnlineService/entities/Annotation.ttl

```diff
@@ -0,0 +1,38 @@
+@prefix ogit.OnlineService: <http://www.purl.org/ogit/OnlineService/> .
+@prefix ogit:                 <http://www.purl.org/ogit/> .
+@prefix dcterms:              <http://purl.org/dc/terms/> .
+@prefix rdfs:                 <http://www.w3.org/2000/01/rdf-schema#> .
+
+ogit.OnlineService:Annotation
+	a rdfs:Class;
+	rdfs:subClassOf ogit:Entity;
+	rdfs:label "Annotation";
+	dcterms:description "Represents an annotation on a OnlineService.";
+	dcterms:valid "start=2024-11-12;";
+	dcterms:creator "Erik Wilfling";
+	ogit:scope "NTO";
+	ogit:parent ogit:Node;
+	ogit:mandatory-attributes (
+		ogit:id
+		ogit:title		# the title of the Anotation
+		ogit:comment	# the comment of the Anotation
+		ogit:status 	# Status of the annotation (Created, Sent, Commented, ReviewRequested, ReviewCompleted, Closed)
+		
+	);
+	ogit:optional-attributes (
+		ogit:createdAt  	# timestamp of creation
+		ogit:jsonContent	# the original content created by teh FrontEnd in JSON format
```

Comment:
IF we introduce jsonContent above then it would need to be referenced as "ogit.OnlineService:jsonContent" here